backend/routes/User_verify.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express")
+const route = express.Router()
+const {User} = require("./model")
+
+route.get("/:id", (req,res)=>{
```

Comment:
This should be a patch route, not a get route because we are modifying the object

Also, rather than making the route /:id, could you make it something like /user/:id/verify

Reply:
I have made the changes and tested it using postman, works fine!